packages/terra-divider/src/terra-dev-site/doc/example/DividerExampleTemplate.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+/* eslint-disable react/prop-types */
+const DividerTemplate = props => (
+  <div>
+    <p>
+      Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industrys standard
+      dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+      book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially
+      unchanged.
```

Comment:
No changes are needed since this is just dummy text. 

I just thought it was a little funny that this isn't Lorem Ipsum text but rather the definition of Lorem Ipsum text. 

https://www.lipsum.com/

Reply:
I hadn’t noticed that. :D I peeled that out of the original test file.